templates/ldap.conf.j2

```diff
@@ -1,6 +1,6 @@
 <LDAP>
     # LDAP server URL
-    URL {{ ldap.url }}
+    URL ldap://{{ ldap.url }}
```

Comment:
You should be able to set the `ldap.url` parameter to include `ldap://` instead of needing to add it here?